Tests/AnalyticsTests/Tests.swift

```diff
@@ -18,9 +18,30 @@ final class Tests: XCTestCase {
         XCTAssertEqual(observer.eventName, ViewEvent.view.name)
         XCTAssertEqual(observer.params, [SourceAnalyticsKey.key: Source.contactList.rawValue])
     }
+    
+    func testLogEventAction() {
+        var values = AnalyticsValues()
+        values[keyPath: \.source] = .contactList
+        var action = AnalyticsAction(values: values)
+
+        action(.view)
+        XCTAssertEqual(observer.eventName, ViewEvent.view.name)
+        XCTAssertEqual(observer.params, [SourceAnalyticsKey.key: Source.contactList.rawValue])
+    }
+    
+    func testLogEventReplacing() {
```

Comment:
Just for consistency:
```suggestion
    func testLogEventReplacingValues() {
```